fluss-server/src/main/java/com/alibaba/fluss/server/kv/prewrite/KvPreWriteBuffer.java

```diff
@@ -221,7 +235,12 @@ public void flush(long exclusiveUpToLogSequenceNumber) throws IOException {
             kvEntryMap.remove(entry.getKey(), entry);
         }
         // flush to underlying kv tablet
-        kvBatchWriter.flush();
+        if (flushedCount > 0) {
+            long start = System.nanoTime();
+            kvBatchWriter.flush();
+            flushCount.inc();
+            flushLatencyHistogram.update((System.nanoTime() - start) / 1_000_000);
```

Comment:
`kvBatchWriter` will hold a buffer, if buffer is full after `put(entry.getKey().key, value.value)` or `kvBatchWriter.delete(entry.getKey().key)`, it'll also flush to rocksdb..

If we calculate the cost of  `kvBatchWriter.flush()`, it'll means the latency of flush to rocksdb instead flushing kvPrewriterBuffer